CRM/Contribute/WorkflowMessage/ContributionTrait.php

```diff
@@ -87,6 +89,7 @@ private function getOrder(): ?CRM_Financial_BAO_Order {
    * Get bool for whether a line item breakdown be displayed.
    *
    * @return bool
+   * @noinspection PhpUnused
```

Comment:
There are several of these added in this PR. Does it turn off "unused variable" IDE warnings? Do we want that?